daemon/api.go

```diff
@@ -533,12 +534,12 @@ func ensureUbuntuCore(chg *state.Change) error {
 		return nil
 	}
 
-	return installSnap(chg, ubuntuCore, "stable", 0)
+	return installSnap(chg, ubuntuCore, "stable", 0, 0)
```

Comment:
Can we please send the user id here too?  Let's take the parameter into ensureUbuntuCore.